clients/fides-js/src/lib/gpp/string-to-consent.ts

```diff
@@ -183,6 +198,14 @@ export const fidesStringToConsent = ({
   cmpApi,
 }: FidesStringToConsentArgs) => {
   const { gpp: gppString }: DecodedFidesString = decodeFidesString(fidesString);
+
+  if (!isValidGppStringFormat(gppString)) {
+    fidesDebugger(
+      "GPP: Invalid GPP string format, skipping consent processing",
+    );
+    return;
+  }
```

Comment:
Seems like this should go below the `if (!fidesString || !gppString || !cmpApi) {` check below